### PR TITLE
feat(prettier-config): Add prettier-config package

### PR DESCRIPTION
**Pull request by lipis**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/prettier-config/index.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "arrowParens": "avoid",
+  "bracketSpacing": false,
+  "endOfLine": "lf",
```
Comment (lipis) [inline packages/prettier-config/index.json]: https://prettier.io/docs/en/options.html#end-of-line

are we OK with `lf`?

Comment (bennyn) [reply packages/prettier-config/index.json]: Yes

File: packages/prettier-config/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@wireapp/prettier-config",
+  "version": "1.0.0",
+  "description": "Wire Prettier config",
+  "main": "index.json",
+  "repository": "git@github.com:wireapp/prettier-config.git",
+  "author": "Wire",
```
Comment (bennyn) [inline packages/prettier-config/package.json]: Author can be removed

File: packages/prettier-config/package.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@wireapp/prettier-config",
+  "version": "1.0.0",
```
Comment (bennyn) [inline packages/prettier-config/package.json]: Let's start at `0.0.1`

File: packages/prettier-config/package.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  "description": "Wire Prettier config",
+  "main": "index.json",
+  "repository": "git@github.com:wireapp/prettier-config.git",
+  "author": "Wire",
+  "license": "MIT"
```
Comment (bennyn) [inline packages/prettier-config/package.json]:
```suggestion
  "license": "GPL-3.0"
```

File: packages/prettier-config/package.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@wireapp/prettier-config",
+  "version": "1.0.0",
+  "description": "Wire Prettier config",
+  "main": "index.json",
+  "repository": "git@github.com:wireapp/prettier-config.git",
```
Comment (bennyn) [inline packages/prettier-config/package.json]:
```suggestion
  "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/prettier-config",
```

**Pull request merged**